sdk/client/src/response.rs

```diff
@@ -65,7 +65,11 @@ pub enum MethodResponse {
     GetCurrencies(Vec<CurrencyInfoView>),
     GetNetworkStatus(u64),
 
+    //
+    // Experimental APIs
+    //
```

Comment:
No need to put this here, because if you add it, it probably won't be removed.  Additionally, what makes it experimental?  If I read this would I know if I could use it?

Reply:
All the proof APIs are technically experimental (they're marked that way in the Jsonrpc docs) so they can have their apis tweaked at any point.